pkg/v1/providers/infrastructure-vsphere/v1.0.1/ytt/base-template.yaml

```diff
@@ -65,7 +65,55 @@ spec:
 apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: VSphereMachineTemplate
 metadata:
-  name: '${ CLUSTER_NAME }-worker'
```

Comment:
well need to potentially disable `VSPHERE_AZ_0` stuff for now, just to get this merged and working for release

Reply:
possible solutions:
- disable VSPHERE_AZs
- enable AZ1/2 in our windows ytt

going w/ one bc...well... *time* :)

Reply:
@stuartpreston fYi were disabling this for expedeince